Code2Gether-Discord-Bot.WebApi/Controllers/ProjectsController.cs

```diff
@@ -165,6 +165,7 @@ public async Task<ActionResult> AddMemberAsync(int projectId, int memberId)
 
             var member = await _dbContext.ProjectMembers
                 .AsAsyncEnumerable()
+                .Where(x => x.ProjectID == projectId)
```

Comment:
This is the fix that was made. Prior to this, it would only check if there was a member with that ID in the join table. This fix also ensures that there isn't a member belonging to a project.